<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(44, 83, 100);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
selection-background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.256545 rgba(15, 32, 39, 255), stop:0.565445 rgba(32, 58, 67, 255), stop:0.801047 rgba(44, 83, 100, 255), stop:1 rgba(15, 32, 39, 255));
</string>
   </property>
   <widget class="QLabel" name="label_length">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Password Length 8 - 64 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_normal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Normal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_medium">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Medium:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Hard:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:#ffffff;
border-radius:8px;
border:1px solid white;
}
QPushButton:hover{
background-color: rgba(212, 212, 212,0.2);
border:2px solid white;
}</string>
    </property>
    <property name="text">
     <string>Generate Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgba(212, 212, 212,0.75);
border-bottom: 1px solid white;
background-color:#000000;
margin-top:12px;
}</string>
    </property>
    <property name="text">
     <string>RANDOM PASSWORD GENERATOR </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius: 8px;
border:1px solid white;
color:white;
padding-left:5px;

}
QLineEdit:focus{
background-color: rgba(212, 212, 212,0.2);
border:2px solid white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:#ffffff;
border-radius:8px;
border:1px solid white;
}
QPushButton:hover{
background-color: rgba(212, 212, 212,0.2);
border:2px solid white;
}</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
border-radius: 4px;
border:1px solid white;
color:white;
padding-left:5px
}
QSpinBox:hover{
background-color: rgba(212, 212, 212,0.2);
border:2px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>751</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.jpg</pixmap>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_normal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_medium">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_hard">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_copy">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>label_length</zorder>
   <zorder>label_normal</zorder>
   <zorder>label_medium</zorder>
   <zorder>label_hard</zorder>
   <zorder>pushButton_generate</zorder>
   <zorder>label_header</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_copy</zorder>
   <zorder>spinBox</zorder>
   <zorder>radioButton_normal</zorder>
   <zorder>radioButton_medium</zorder>
   <zorder>radioButton_hard</zorder>
   <zorder>label_copy</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
